app-devel/mimalloc/autobuild/defines

```diff
@@ -0,0 +1,6 @@
+PKGNAME=mimalloc
+PKGDES="A compact general purpose allocator with excellent performance"
```

Comment:
A high-performance general purpose allocator.

Avoid fluff/advertisement from the upstream.